README.md

```diff
@@ -14,64 +14,93 @@ https://www.taniarascia.com/make-a-static-website-with-jekyll/
 
 ## Set up Discourse Forum
 
-### Set up Compute Canada instance
-
-- Create instance on Compute Canada
-  - https://graham.cloud.computecanada.ca/dashboard/project/instances/
+### Reference
+https://github.com/discourse/discourse/blob/master/docs/INSTALL-cloud.md
 
+Create account & droplet in Digital Ocean. Droplet configure : 1GB RAM, 1 vCPU,25 GB HDD,	1 TB transfer.
 
-- SSH permissions: add the "SSH" ingress rule to allow ssh into the instances:
+### Setup email
+  - Create an account on [SendGrid](https://sendgrid.com/)
+  - Generate and copy to clipboard an API Key in SendGrid/Settings/API Keys
 
-To connect to server:
+### Setup Discourse server
+Connect to the droplet server provided by Digital Ocean, then do:
+  * Install Docker:
 ~~~
-ssh ubuntu@X.X.X.X  # connect to floating IP
+wget -qO- https://get.docker.com/ | sh
 ~~~
-
-### Set up discourse
-https://github.com/discourse/discourse/blob/master/docs/INSTALL-cloud.md
-
-Connect to server, then do:
+- Clone Discourse deploy
 ~~~
-sudo su
 mkdir /var/discourse
+git clone https://github.com/discourse/discourse_docker.git /var/discourse
 cd /var/discourse
-git clone https://github.com/discourse/discourse_docker.git .
 ~~~
-
-- add valid SPF and DKIM records in your DNS:
-
-- Create DNS records for github-page and discourse forum server:
-SCREENSHOTS: namecheap_add-records
-
-Setup Discourse install:
+- Install Discourse
 ~~~
 ./discourse-setup
 Hostname      : forum.spinalcordmri.org
-Email         : xxx@spinalcordmri.org
-SMTP address  : smtp.elasticemail.com
-SMTP port     : 2525
-SMTP username : xxx  # get this from Elasticmail > Settings > SMTP/API
-SMTP password : pa$$word  # get this from Elasticmail > Settings > SMTP/API
+Email         : spinalcordmri@gmail.com
+SMTP address  : smtp.sendgrid.net
+SMTP port     : 587
+SMTP username : apikey
+SMTP password : [SendGrid API Key]
 Let's Encrypt : [press Enter]
 ~~~
-
-Launch
-~~~
-sudo ./launcher bootstrap app
-~~~
+### Setup subdomain in namecheap
+  - To create a subdomain, please do the following:
+    - Go to your Domain List and click Manage next to the domain
+    - Select the Advanced DNS tab
+    - Find the Host Records section and click on the Add New Record button
+    - Select A Record for Type and enter the Host `forum.spinalcordmri.org`  you would like to point to an IP address `DigitalOcean_Server_IP_Address`
+
+### Configuring Google login for Discourse ([reference](https://meta.discourse.org/t/configuring-google-login-for-discourse/15858))
+
+Using the spinalcordmri@gmail.com account, go to https://console.developers.google.com, click on Credentials and create a new Project.
```

Comment:
same comment about the email address

Reply:
Fixed in https://github.com/spinalcordmri/spinalcordmri.github.io/pull/15/commits/d555da77237e377e95765083bbac94596eeb3a76